test/libsolidity/smtCheckerTests/operators/unary_add_array.sol

```diff
@@ -15,4 +15,4 @@ contract C
 	}
 }
 // ----
-// Warning 6328: (240-253): CHC: Assertion violation happens here.\nCounterexample:\narray = []\nx = 38\n\n\nTransaction trace:\nconstructor()\nState: array = []\nf(38)
+// Warning 6328: (240-253): CHC: Assertion violation happens here.
```

Comment:
This does not report CEX anymore, probably because the second call to Spacer, to get the CEX, runs into the resource limit.